### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by anyepeter**
# Styling 
The entire application is styled with the following requirements 📑 

### General requirements
✅ Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
✅ Make sure that you used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
✅ Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
### HTML/CSS requirements
✅ Follow our list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
### Project requirements
✅ You will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/).
✅[Login to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md) of the project).
✅Open the project Bookstore CMS.
✅Add any missing JSX elements to your projects (without adding extra functionality)
✅You may use the CSS rules detailed in Zeplin to style the website.
✅ Your final design should match Zeplin's design (this includes all elements + the same styling)

### CODE REVIEW

File: src/CSSmodule/bookitem.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.detail {
+  display: flex;
+  flex-direction: column;
+}
+
+.titles {
```
Comment (AnselemOdims) [inline src/CSSmodule/bookitem.css]: - [ ] Good job working on the styling, but please kindly ensure that the circular progress is the same size with that of the original design

### Your Design
image

### Original Design
image

File: src/CSSmodule/bookitem.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  font-size: 42px;
+}
+
+.category {
+  color: rgb(58, 58, 59);
+}
+
+.author {
+  color: #227ded;
+}
```
Comment (AnselemOdims) [inline src/CSSmodule/bookitem.css]: - [ ] Please kindly ensure that you stick to the design guidelines(font weights, font-size, spacing, font type, etc)

### Your design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @anyepeter , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @anyepeter , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
*Check the comment below*

- [ ] Your design is missing the vertical gray colored line, try improve on this if you got some spare time
- [ ] The circular image size is still big as compared to the original. please, let's also work on this if we got some spare time to improve our design.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**